simulators/xbeach/v1.23/Dockerfile

```diff
@@ -1,84 +1,36 @@
-FROM library/centos:7 as test_env
+FROM inductiva/kutu:base-image_v0.1.0 as test_env
 
-RUN yum update -y && \
-	yum install -y wget \
-    unzip &&\
-    wget https://storage.googleapis.com/inductiva-api-demo-files/xbeach-input-example.zip -P /home/ && \
-    unzip /home/xbeach-input-example.zip -d /home/ && \
-    rm /home/xbeach-input-example.zip
+RUN wget https://storage.googleapis.com/inductiva-api-demo-files/xbeach-input-example.zip -P /home/ && \
+	unzip /home/xbeach-input-example.zip -d /home/ && \
+	rm /home/xbeach-input-example.zip
 
 COPY ./test_sim.sh /home/test_sim.sh
-RUN chmod +x /home/test_sim.sh
-
-
-FROM library/centos:7 as builder
 
-RUN yum update -y && \
-	yum install epel-release -y
+RUN chmod +x /home/test_sim.sh
 
-RUN	yum install -y gcc \
-	hdf5-devel \
-	libtool \
-	make \
-	netcdf-devel \
-	netcdf-fortran \
-	netcdf-fortran-devel \
-	openmpi-devel \
-	git \
-	python-mako
+FROM inductiva/kutu:base-image_v0.1.0 as build
 
-ENV PATH=/usr/lib64/openmpi/bin:$PATH \
-	LD_LIBRARY_PATH=/usr/lib64/openmpi/lib:$LD_LIBRARY_PATH
+RUN apt update -qq && \
+    apt install --no-install-recommends -y \
+        subversion \
+        python3 \
+        python3-mako \
+		netcdf-bin \
+        libnetcdf-dev \
+		libnetcdff-dev  && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    apt clean
 
-RUN git clone https://github.com/openearth/xbeach.git /xbeach
+RUN svn co https://svn.oss.deltares.nl/repos/xbeach/trunk /xbeach
```

Comment:
How do we specify the version here? If they update the code, we would be building a newer version of XBeach but labeling it as 1.23, no?

Reply:
We can use https://svn.oss.deltares.nl/repos/xbeach/tags/ to download